Sources/Plasma/PubUtilLib/plPipeline/DX/plDXSettings.h

```diff
@@ -74,75 +74,16 @@ class plDXDeviceRef;
 class plDXVertexBufferRef;
 class plDXIndexBufferRef;
 
-class plDXViewSettings
-{
-public:
-    uint32_t                  fRenderState;
-
-    plRenderRequest*        fRenderRequest;
-
-    uint32_t                  fDrawableTypeMask;
-    uint32_t                  fSubDrawableTypeMask;
-
-    DWORD                   fClearColor;
-    float                   fClearDepth;
-
-    plFogEnvironment        fDefaultFog;
-
-    plCullTree              fCullTree;
-    bool                    fCullTreeDirty;
-    uint16_t                  fCullMaxNodes;
-
-    enum XformResets
-    {
-        kResetProjection    = 0x01,
-        kResetCamera        = 0x02,
-        kResetL2W           = 0x04,
-
-        kResetAll           = 0x07
-    };
-
-    uint8_t                   fXformResetFlags;
-    bool                    fLocalToWorldLeftHanded;
-    bool                    fWorldToCamLeftHanded;
-
-    mutable hsVector3       fDirection;
-    mutable hsVector3       fUp;
-    mutable hsVector3       fAcross;
-    hsPoint3                fWorldPos;
-
-    mutable bool            fViewVectorsDirty;
-
-    hsMatrix44              fLocalToWorld;
-    hsMatrix44              fWorldToLocal;
-
-    const hsMatrix44&       GetLocalToWorld() const { return fLocalToWorld; }
-    const hsMatrix44&       GetWorldToLocal() const { return fWorldToLocal; }
-
-    plViewTransform         fTransform;
-
-    const hsMatrix44&       GetWorldToCamera() const { return fTransform.GetWorldToCamera(); }
-    const hsMatrix44&       GetCameraToWorld() const { return fTransform.GetCameraToWorld(); }
-    bool                    IsPerspective() const { return fTransform.GetPerspective(); }
-
-    void            Reset();
-};
-
 class plDXGeneralSettings
 {
     public:
 
         bool                    fFullscreen;
-        hsWinRef                fHWnd;
-        uint32_t                  fColorDepth;
+        //hsWinRef                fHWnd;
```

Comment:
Seems these are inside of pl3DPipeline.h now?  Are they still needed here as comments?